Menu.java

```diff
@@ -0,0 +1,86 @@
+import javax.swing.*;
+
+import java.awt.*;
+import java.awt.event.*;
+
+public class Menu 
+{
+    int difficulty;
+    Game game;
+    
+    public static void main(String[] args) 
+    {
+        Menu m = new Menu();
+    }
+
+    public Menu()
```

Comment:
can be simplified to function that gets a JRadioButton and list of actions to apply to it